examples/camera-app/linux/src/clusters/webrtc-provider/webrtc-provider-manager.cpp

```diff
@@ -440,6 +445,92 @@ WebRTCProviderManager::ValidateStreamUsage(StreamUsageEnum streamUsage,
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR WebRTCProviderManager::ValidateVideoStreamID(uint16_t videoStreamId)
+{
```

Comment:
This should call into a CameraAVSM Delegate method that does the validation.

Reply:
Done